lib/note-toolbar/index.tsx

```diff
@@ -134,7 +138,7 @@ export class NoteToolbar extends Component<Props> {
           <div className="note-toolbar__button">
             <IconButton
               icon={<InfoIcon />}
-              onClick={this.props.onShowNoteInfo}
+              onClick={() => onToggleNoteInfo()}
```

Comment:
nitpick: we don't need to create a new function here to wrap the existing one. we can pass `onToggleNoteInfo` directly into it and that will save us down the line (since as-is here we're creating a new function on every call to `render()`)

```ts
onClick={onToggleNoteInfo}
```

note that in some cases this isn't possible because the calling function supplies arguments we don't want or expect. in those cases we need to manually prevent that by creating the wrapper. technically you've done that in `mapDispatchToProps` when you wrote `() => dispatch(toggleNoteInfo(true))` because any arguments passed by `onClick` would be ignored there.